.github/workflows/ci.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         with:
           toolchain: stable
           components: clippy
-      - run: cargo clippy -- -D warnings -D clippy::panic -D clippy::todo
+      - run: cargo clippy -- -D clippy::panic -D clippy::todo
```

Comment:
Since we don't have Rust veterans in the team I would encourage to maintain clippy failing on warnings, we need more learning opportunities

Reply:
Ok @7flying I understand. These warnings are related to the older format string syntax usage in rs files

I.e. https://github.com/fdo-rs/fido-device-onboard-rs/blob/main/util/src/servers/mod.rs#L73 
`format!("{}.yml", guid)`
Instead of 
`format!("{guid}.yml")`

There is more than one file using this syntax. Thus, should we change all of them (despite those changes wouldn't be related to Clippy command) in a separated PR?